.github/actions/build-and-push-to-quay/action.yml

```diff
@@ -85,6 +85,16 @@ runs:
         TAGS="$TAGS ${SHORT_SHA}"
 
         echo "TAGS=$TAGS" >> $GITHUB_OUTPUT
+    - name: Run pre-build hooks
+      shell: bash
+      run: |
+        cd ${{ inputs.docker_directory }}
+        if [[ -f pre-build-hook.sh ]]; then
+          echo "detected pre-build hook, running"
+          /bin/bash pre-build-hook.sh
+        else
+          echo "no pre build hook detected"
+        fi
```

Comment:
To get the extension installed into the postgres container, we have a script that builds the extension, and puts the package into a directory "extensions" in the postgres directory before doing the build of the postgres image.

I think we ought to instead publish our packages and just pull them in in the docker build over the internet from our repository instead of doing this

Reply:
publish the pgx extension as a crate, then pull from crates.io and install?

as opposed to cloning the repo, then pushing to crates.io and quay.io at same time (or in sequence)?

Reply:
I mean publish to an APT repository the .deb package. Seems to me like that's the best way to package and distribute extensions.

Reply:
Like I mean i'm imagining our workflow will build and publish to our APT repository all our extensions (and I think we don't need to publish extensions to crates.io). So that's another artifact type.